frontend/src/layout/navigation-3000/navigationLogic.tsx

```diff
@@ -515,21 +515,20 @@ export const navigation3000Logic = kea<navigation3000LogicType>([
                                   to: urls.earlyAccessFeatures(),
                               }
                             : null,
-                        {
-                            identifier: Scene.DataWarehouse,
-                            label: 'Data warehouse',
-                            icon: <IconDatabase />,
-                            to: isUsingSidebar ? undefined : urls.dataWarehouse(),
-                        },
                         featureFlags[FEATURE_FLAGS.SQL_EDITOR]
                             ? {
                                   identifier: Scene.SQLEditor,
-                                  label: 'SQL Editor',
+                                  label: 'Data warehouse',
```

Comment:
Thought we wanna call it "SQL Editor" and not data warehouse?

Reply:
slightly worried people will get lost if we swap completely